### PR TITLE
📕 Add new translations for the security project

### LINKED ISSUE #7332: New Copy Translation Request - Password security criteria
**Target Languages:** 
PT
JP
FR

**New Copies:**
- Include 8 or more characters, with at least one uppercase letter, lowercase letter, and a number.
- One more try before account gets locked for 5 minutes.
- Account is locked for 5 minutes. Account owner has been notified.
- Incorrect email or password.

~~**EDIT - adding another string**
Set a secure password~~
It turns out 'Set a secure password' won't be needed after all

### PR DESCRIPTION
**Pull request by pdrj**
## What's this?
This adds new translations for the security project and renames one old string.

### Relationships

Closes #7332 

## How is it done?
By utilizing native speakers.

## Review hints
You can make sure that the security indicator copy displays properly on your device using the pt/jp/fr/en locales. For the other strings, please validate they are right, if you are a speaker of the language. Otherwise, you can verify I copied them over correctly from the issue.

## Localizations
Yes

## :squid: Permissions
Anyone

### CODE REVIEW

File: Toggl.Shared/Resources.fr.resx [hunk 1]
```diff
@@ -1226,6 +1226,22 @@ des informations ci-dessous.</value>
     <data name="InvalidPasswordError" xml:space="preserve">
         <value>Le mot de passe doit faire au moins 6 caractères</value>
     </data>
+    <data name="StrongPasswordCriteria" xml:space="preserve">
+        <value>Inclure 8 caractères minimum, avec au moins une majuscule, une minuscule, et un chiffre.</value>
+        <comment>Displayed under the password field to let the user know what criteria must be met when setting a password</comment>
```
Comment (daividssilverio) [inline Toggl.Shared/Resources.fr.resx:1231]: We don't add the comments to the translations.

File: Toggl.Shared/Resources.ja.resx [hunk 2]
```diff
@@ -138,14 +138,30 @@
...
     <data name="InvalidEmailOrPassword" xml:space="preserve">
         <value>メール/パスワードは無効です。
 もう一度お試しください</value>
     </data>
+    <data name="StrongPasswordCriteria" xml:space="preserve">
+        <value>8文字以上で大文字・小文字・数字を最低1回ずつ使用してください。</value>
+        <comment>Displayed under the password field to let the user know what criteria must be met when setting a password</comment>
+    </data>
+    <data name="IncorrectEmailOrPassword" xml:space="preserve">
+        <value>メールかパスワードが間違っている。</value>
```
Comment (daividssilverio) [inline Toggl.Shared/Resources.ja.resx:154]: This is the reviewed version.
```suggestion
        <value>メールまたはパスワードが間違っています。</value>
```

File: Toggl.Shared/Resources.ja.resx [hunk 3]
```diff
@@ -138,14 +138,30 @@
...
+        <comment>Displayed when signin couldn't be completed with the given credentials</comment>
+    </data>
+    <data name="IncorrectEmailOrPasswordOneMoreTryBeforeAccountLocked" xml:space="preserve">
+        <value>メールかパスワードが間違っている。 もう一回間違っていたら、アカウントが五分間までにロックになります。</value>
```
Comment (daividssilverio) [inline Toggl.Shared/Resources.ja.resx:158]: These are the right reviewed versions.
```suggestion
        <value>メールまたはパスワードが間違っています。もう一度間違えると5分間アカウントにロックがかかります。</value>
```

File: Toggl.Shared/Resources.ja.resx [hunk 4]
```diff
@@ -138,14 +138,30 @@
...
+        <comment>Displayed when signin couldn't be completed with the given credentials and if there's one more failed attempt, account will be locked</comment>
+    </data>
+    <data name="IncorrectEmailOrPasswordAccountIsLocked" xml:space="preserve">
+        <value>メールかパスワードが間違っている。 アカウントがロックされました。アカウント持ち主が連絡されました。</value>
```
Comment (daividssilverio) [inline Toggl.Shared/Resources.ja.resx:162]: Reviewed versions:
```suggestion
        <value>メールまたはパスワードが間違っています。アカウントがロックされました。アカウントの持ち主に通知されました。</value>
```

File: Toggl.Shared/Resources.ja.resx (hunk 2 above)
Comment (pdrj) [reply Toggl.Shared/Resources.ja.resx:154]: Sorry, I copied from the wrong comment 🙈

**Pull request merged**